### PR TITLE
:fire: Fixed some typos

### PR DESCRIPTION
**Pull request by Inffinite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ bun is a new:
 - JavaScript/TypeScript/JSX transpiler
 - JavaScript & CSS bundler
 - Task runner for package.json scripts
-- npm-compatible package manager
+- Npm-compatible package manager
```
Comment (icarusgk) [inline README.md:12]: I think "npm" is used as a proper noun in here which itself starts with a lowercase "n"

File: README.md [hunk 2]
```diff
@@ -3193,7 +3193,7 @@ You’ll want to make sure `zig` is in `$PATH`. The specific version of Zig expe
 
 #### Build bun (macOS)
 
-If you’re building on an Apple Silicon device, you’ll need to do is ensure you have set an environment variable `CODESIGN_IDENTITY`. You can find the correct value by visiting `Keychain Access` and looking under your `login` profile for `Certificates`. The name would usually look like `Apple Development: user@example.com (WDYABC123)`
+If you’re building on an Apple Silicon device, what you’ll need to do is ensure you have set an environment variable `CODESIGN_IDENTITY`. You can find the correct value by visiting `Keychain Access` and looking under your `login` profile for `Certificates`. The name would usually look like `Apple Development: user@example.com (WDYABC123)`
```
Comment (icarusgk) [inline README.md:3196]: The sentence without "what" sounds more fluent and natural.

Comment (Ahummeling) [inline README.md:3196]: Then it should be `you'll need to ensure you have...` leaving out `do is`. In the current text, it seems like the word `what` is simply missing. Rewriting to `` you'll need to ensure you have set an environment variable `CODESIGN_IDENTITY` ``, will sound more fluent and natural as mentioned.

File: README.md (hunk 1 above)
Comment (merlinseela) [reply README.md:12]: i dont think this is a correct change.
while "npm" is a name of a "thing" and therefore a noun it is still in small letters. 

"npm" is the correct way to write it. If you look on the official website it is written with 3 small letters everywhere. And i think thoose guys that make the thing know how to name it. :)

File: README.md (hunk 2 above)
Comment (merlinseela) [reply README.md:3196]: @Ahummeling is correct.

File: README.md (hunk 1 above)
Comment (icarusgk) [reply README.md:12]: That's exactly what I'm saying? I don't know how I came across, but I'm sure that those guys that make the thing know how to name it too! 🤠 As "npm".

File: README.md (hunk 2 above)
Comment (icarusgk) [reply README.md:3196]: @Ahummeling You're right, sounds better rewriting it that way.

**Review by jub0t — APPROVED** [review]

**Pull request closed without merging**